core/src/main/java/io/questdb/cairo/MetadataCache.java

```diff
@@ -192,25 +194,23 @@ private void hydrateTableStartup(@NotNull TableToken token, boolean throwError)
                     .$(", version=").$(metadataVersion)
                     .I$();
 
+            boolean isMetaFormatUpToDate = TableUtils.isMetaFormatUpToDate(
```

Comment:
i think this method always used in context of reading metadata file, may be we can change its signature to reflect that write a bunch of tests?